packages/material-ui-system/src/Box/Box.spec.tsx

```diff
@@ -36,7 +36,7 @@ function GapTest() {
       display: 'flex',
       alignItems: 'center',
       flex: '1 0',
-      gridGap: (theme) => theme.spacing(2),
+      gridGap: 16,
```

Comment:
We might want to rename the CSS property, as deprecated

```suggestion
      gap: '16px',
```